cmd/realm/trigger_policy.go

```diff
@@ -0,0 +1,189 @@
+// Copyright © 2019 Ispirata Srl
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package realm
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/astarte-platform/astartectl/utils"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+// triggersPoliciesCmd represents the triggers command
+var triggersPoliciesCmd = &cobra.Command{
+	Use:     "trigger-policy",
```

Comment:
Naming consistency: other commands prefer the plural and set the singular as alias (e.g. `interfaces` and `interface`)
```suggestion
	Use:     "trigger-policies",
```